web/src/views/DandisetLandingView/ContactDialog.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <v-menu
+    offset-y
+    left
+  >
+    <template
+      #activator="{ on, attrs }"
+    >
+      <v-btn
+        id="download"
+        outlined
+        block
+        v-bind="attrs"
+        v-on="on"
+      >
+        <v-icon
+          color="primary"
+          left
+        >
+          mdi-card-account-mail
+        </v-icon>
+        <span>Contact</span>
+        <v-spacer />
+        <v-icon right>
+          mdi-chevron-down
+        </v-icon>
+      </v-btn>
+    </template>
+    <v-card>
+      <v-card-title class="pb-0">
+        Select an e-mail Recipient
+      </v-card-title>
+      <v-list>
+        <v-tooltip
+          :disabled="loggedIn()"
+          open-on-hover
+          right
+        >
+          <template #activator="{ on }">
+            <v-list-item
+              v-on="on"
+              :class="loggedIn() ? 'black--text' : 'grey--text'"
+              :selectable="!loggedIn()"
+              :href="`mailto:${owners}?subject=Regarding%20Dandiset%20${currentDandiset?.name}%20${currentDandiset?.metadata?.id} &body=${currentUser?.value?.username}`"
```

Comment:
This should be refactored from here and the other place it appears into a computed property of the component. That will furthermore enable you to deal with possible missing data (all the optional chaining operators that appear) in a more robust way (either raising an error or supplying values besides `undefined` to use), as well as improve the readability of this expression.

Reply:
While you're at it, I think it's odd to have the body filled it with my own email address. I assume that can be left out and you'll just get your email client with a fresh, blank email body?

Reply:
changed in d9847970259f7fafb446622621c4ae9ee8b30e6d